### ISSUE TITLE
Makefile `mkdir` needs `-p` flag

### ISSUE THREAD

**Issue by toastal** [sass/libsass#1433]
I tried to build libsass this morning on Debian and `mkdir` was bawking on creating directory because it's exists. If it contained the `-p` flag will prevent that error.

**Comment by xzyfer** [comment]
This has come up before. The concern has been that `-p` is not portable. See discussion in https://github.com/sass/libsass/pull/795

We're open to workarounds.

**Comment by saper** [comment]
We can  use `-$(MKDIR)/lib`

**Comment by xzyfer** [comment]
@toastal can you be more specific on where it failed? I was not able to produce this on a fresh checkout and `make` of libsass master.

**Comment by saper** [comment]
> @toastal can you be more specific on where it failed? I was not able
> to produce this on a fresh checkout and `make` of libsass master.

It might be related to the fact that previous release didn't clean "lib" folder.
So it might happen only once after upgrade or something like that.

**Comment by toastal** [comment]
The `-p` flag is for after files are created so of course it didn't fire on a fresh checkout. The steps I'd done were `git pull` that was behind a month or two. When I ran `make` (without `make clean` so that I'm not having to rebuild what's not needed), make bawked on creating one of the folders because it existed and so nothing was built. I didn't realize that this is an unfortunate incompatibility with Windows's `mkdir`. I manually added `-p` to the Makefile on line 5 and it has worked ever since.
